Bluetooth: tests: shell: Remove config not supported on native_posix

joerchan opened the pull request:
Remove CONFIG_CONSOLE_HANDLER from project configuration. When building
for native_posix this produces the following warning:

warning: CONSOLE_HANDLER was assigned the value 'y' but gotthe value 'n'
Check these unsatisfied dependencies: SERIAL_SUPPORT_INTERRUPT (=n)

The shell works without this option since this is handled by the shell
itself instead.

Signed-off-by: Joakim Andersson <joakim.andersson@nordicsemi.no>

joerchan committed:
Bluetooth: tests: shell: Remove config not supported on native_posix

Remove CONFIG_CONSOLE_HANDLER from project configuration. When building
for native_posix this produces the following warning:

warning: CONSOLE_HANDLER was assigned the value 'y' but gotthe value 'n'
Check these unsatisfied dependencies: SERIAL_SUPPORT_INTERRUPT (=n)

The shell works without this option since this is handled by the shell
itself instead.

Signed-off-by: Joakim Andersson <joakim.andersson@nordicsemi.no>

jhedberg reviewed (changes requested):
while you're at it, please remove it from prj_br.conf as well as mesh_shell/prj.conf in the same go, thanks :)

joerchan committed:
Bluetooth: tests: shell: Add missing dependency in mesh.conf

Add missing dependency in mesh.conf.
Handles this warning: Warning: BT_MESH_SHELL was assigned the value 'y'
but got the value 'n'. Check these unsatisfied dependencies:
BT_MESH_HEALTH_CLI (=n)

Signed-off-by: Joakim Andersson <joakim.andersson@nordicsemi.no>

joerchan commented:
> while you're at it, please remove it from prj_br.conf as well as mesh_shell/prj.conf in the same go, thanks :)

Fixed.

Also updated mesh.conf, but I'm wondering if this conf file is needed since there is a mesh_shell in addition? Is this an old conf file from before the mesh_shell was added and could be removed?

jhedberg commented:
> Is this an old conf file from before the mesh_shell was added and could be removed?

I suspect it can be removed, but let's do that in a separate PR for 2.3 (this one can go into 2.2, I think)
